### DISCUSSION TITLE
Powód odrzucenia widoczności oferty na rynku

### DISCUSSION THREAD

**Discussion by LukaszWypych** [allegro/allegro-api#7125] [Suggestions - resolved]
Przy pobieraniu ofert dostajemy status jej widoczności w serwisie allegro-cz. W przypadku REJECTED dobrze byłoby otrzymać informację z powodem, czemu została odrzucona.

CASE: Przy testowaniu na sandboxie, ustawiłem ofertę zgodnie z zaleceniami dokumentacji i w dalszym ciągu otrzymywałem REJECTED i nie miałem informacji czemu. Dopiero po wejściu w ofert przez stronę zobaczyłem informację, że cena czeska nie może się różnić od ceny polskiej o ponad 10% (Co przy okazji też dobrze byłoby napisać w dokumentacji).

**Comment by AureliuszBrussy** [comment]
Powód odrzucenia zwracamy na endpointcie do pobierania zdarzeń ofertowych ([GET /sale/offer-events](https://developer.allegro.pl/documentation#operation/getOfferEvents)). Natomiast rozumiem, że chciałbyś, żeby informacja była zwracana także w modelu samej oferty (np. po odpytaniu GET /sale/product-offers)?

**Reply by PhotoSoft** [reply]
Zdecydowanie jest potrzebna taka informacja w modelu oferty.

**Comment by LukaszWypych** [comment]
Tak

**Comment by AureliuszBrussy** [comment]
Dziękuję za informację, oczywiście bierzemy taką sugestię pod uwagę. Jeżeli coś zmienimy - oczywiście będziemy to komunikować.

**Comment by MartaNowaczyk** [comment]
**UPDATE:** Zespół planuje wprowadzić taką zmianę, jednak zmiana ta może pojawić się najwcześniej w ostatnim kwartale tego roku.

**Comment by MartaNowaczyk** [comment]
**UPDATE:** Zmiana jest już od pewnego czasu dostępna. Zmieniłam status na "Suggestions - resolved".
